.github/actions/compute-sha256/src/main.ts

```diff
@@ -0,0 +1,21 @@
+import * as core from "@actions/core";
+import * as fs from "fs";
+import * as crypto from "crypto";
+
+function shasum256(untrustedPath: string): string {
+  if (!fs.existsSync(untrustedPath)) {
+    throw new Error(`File ${untrustedPath} not present`);
+  }
+  const untrustedFile = fs.readFileSync(untrustedPath);
+  return crypto.createHash("sha256").update(untrustedFile).digest("hex");
+}
+
+async function run(): Promise<void> {
```

Comment:
This function doesn't seem to use 'await' or anything so it can probably just be a regular 'function' rather than an 'async function'.

Reply:
I have updated it.